project/plugins.sbt

```diff
@@ -7,7 +7,7 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.5") // when updating major version, also update play-circe version
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9") // when updating major version, also update play-circe version
```

Comment:
this is only a patch update.  The comment only applies to the first two numbers e.g. 2.7->2.8